src/scene/scene_forwardrenderer.js

```diff
@@ -631,7 +636,31 @@ pc.extend(pc, function () {
             drawCalls.sort(sortDrawCalls);
 
             // Render a depth target if the camera has one assigned
-            if (camera._depthTarget) {
+            if (camera._renderDepthRequests) {

```

Comment:
The point of this is communication of API though. If I see a method that is documented, then I know I'm safe to use it. If I'm searching for properties with a leading underscore then I know I shouldn't use it.

Which is this? Should users of the camera node have access to this property or is it secret only for internal use.

It looks like authors of post effects will need to use it.